stable/aws-for-fluent-bit/README.md

```diff
@@ -49,6 +49,7 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `cloudWatch.logStreamName` | The name of the CloudWatch Log Stream that you want log records sent to. |  |
 | `cloudWatch.logStreamPrefix` | Prefix for the Log Stream name. The tag is appended to the prefix to construct the full log stream name. Not compatible with the log_stream_name option. | `"fluentbit-"` |
 | `cloudWatch.logKey` | By default, the whole log record will be sent to CloudWatch. If you specify a key name with this option, then only the value of that key will be sent to CloudWatch. For example, if you are using the Fluentd Docker log driver, you can specify logKey log and only the log message will be sent to CloudWatch. |  |
+| `cloudWatch.logRetentionDays` | If set to a number greater than zero, and newly create log group's retention policy is set to this many days. | |
```

Comment:
Can you please update the [values.yaml file](https://github.com/aws/eks-charts/blob/12d2ba5ace78a5cf62bbcf2e5ab97c58f62f0c69/stable/aws-for-fluent-bit/values.yaml#L53-L65) with this new variable?